lib/v2/sanlian/channel.js

```diff
@@ -0,0 +1,42 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const util = require('@/v2/nintendo/utils');
+
+const rootUrl = 'https://www.lifeweek.com.cn/column';
+const articleRootUrl = 'https://www.lifeweek.com.cn/article';
+module.exports = async (ctx) => {
+    const channel = ctx.params.id;
+    const url = `${rootUrl}/${channel}`;
+    const response = await got(url, {
+        responseType: 'buffer',
+    });
+    const result = await util.nuxtReader(response.data);
+    if (!result.articleList) {
+        throw new Error('新闻信息不存在，请报告这个问题');
+    }
```

Comment:
Please use the API endpoint of `/api/userWebFollow/getFollowTagContentList` instead of looking at the HTML.
docs: https://docs.rsshub.app/joinus/new-rss/before-start#acquire-data